src/Custom-hook/user-api.js

```diff
@@ -1,21 +1,21 @@
-import axios from "axios";
-import { useEffect, useState } from "react";
+// import axios from "axios";
+// import { useEffect, useState } from "react";
 
-export const useAxios = () => {
-  const [data, setData] = useState([]);
-  const [loader, setLoader] = useState(false);
+// export const useAxios = () => {
+//   const [data, setData] = useState([]);
+//   const [loader, setLoader] = useState(false);
 
-  useEffect(() => {
-    (async () => {
-      try {
-        setLoader(true);
-        const { data } = await axios.get("/api/users");
-        setLoader(false);
-        setData(data.users);
-      } catch (error) {
-        console.log(error);
-      }
-    })();
-  }, []);
-  return { data, loader };
-};
+//   useEffect(() => {
+//     (async () => {
+//       try {
+//         setLoader(true);
+//         const { data } = await axios.get("/api/users");
+//         setLoader(false);
+//         setData(data.users);
+//       } catch (error) {
+//         console.log(error);
+//       }
+//     })();
+//   }, []);
+//   return { data, loader };
+// };
```

Comment:
Should remove unwanted comments in this file

Reply:
ok